browser/resources/brave_new_tab_page_refresh/components/background/background.tsx

```diff
@@ -71,17 +71,27 @@ function setBackgroundVariable(value: string) {
 }
 
 function ImageBackground(props: { url: string, onLoadError?: () => void }) {
+  const [blobUrl, setBlobUrl] = React.useState<string | null>(null)
```

Comment:
Curious: do we need to keep the `blobUrl` in component state, or could it just be  a local variable in the `useEffect` callback?